CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 This file is used to list changes made in each version of the devopsbox cookbook.
 
+## Master
+ * install ncdu package.
+
```

Comment:
Haven't really thought about that, to be honest.  IMHO, we should bump versions and use Git tags on Master whenever we make a changes, regardless, as failing to do so could break things for cookbook users who are pulling directly via GitHub/Berksfile.